### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by nhnt11**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* global require, module, __dirname */
+
+const path = require("path");
+
+module.exports = {
+  entry: {
+    feature: "./feature.js/index.js",
```
Comment (englehardt) [inline webpack.config.js]: It seems strange to me to have folders with file extensions in their name. Is this a webpack design pattern?

File: feature.js/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import {
+  CookieInstrument,
+  JavascriptInstrument,
+  HttpInstrument,
+  NavigationInstrument,
+} from "openwpm-webext-instrumentation";
+
+import * as loggingDB from "./loggingdb.js";
+
+async function foo() {
```
Comment (englehardt) [inline feature.js/index.js]: can we call this `main` or something similar? `foo` is typically used for example code.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  "keywords": [
+    "firefox",
+    "mozilla",
+    "openwpm"
+  ],
+  "license": "MPL-2.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nhnt11/openwpm-firefox-webext"
```
Comment (englehardt) [inline package.json]: Update to `https://github.com/mozilla/openwpm-firefox-webext/`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  },
+  "scripts": {
+    "prebuild": "cp -r node_modules/WEE-Sockets-API/lib/* src/ && cp -r node_modules/WEE-ProfileDirIO-API/lib/privileged/* src/privileged/ && webpack",
+    "build": "web-ext build",
+    "eslint": "eslint . --ext jsm,js,json",
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "npm run eslint",
+    "lint:web-ext-lint": "web-ext lint",
+    "sign": "echo 'TBD, see: https://bugzilla.mozilla.org/show_bug.cgi?id=1407757'",
```
Comment (englehardt) [inline package.json]: Not sure if this bug is relevant?

**Comment by englehardt** [comment]
`visit_id` is always -1 in the `2019-01-23_alexatop1m` test crawl. This implies that it either isn't being sent or isn't being received correctly. We should investigate the cause.

File: webpack.config.js (hunk 1 above)
Comment (nhnt11) [reply webpack.config.js]: Yeah, I believe so. All the files inside the folder are "packed" into feature.js. I got the pattern from @motin's repos :)

**Comment by nhnt11** [comment]
> `visit_id` is always -1 in the `2019-01-23_alexatop1m` test crawl. This implies that it either isn't being sent or isn't being received correctly. We should investigate the cause.

Filed #8

**Comment by nhnt11** [comment]
I addressed all your comments. Closing this PR; we can work on "proper" PRs now that initial review is done.

**Pull request closed without merging**